tests/validation/tests/v3_api/test_bkp_restore_s3_recover.py

```diff
@@ -0,0 +1,158 @@
+import pytest
+from .common import *  # NOQA
+from .test_rke_cluster_provisioning import rke_config, \
+    create_and_validate_custom_host, node_template_ec2
+
+namespace = {"p_client": None, "ns": None, "cluster": None, "project": None,
+             "nodes": []}
+backup_info = {"backupname": None, "backup_id": None, "workload": None,
+               "backupfilename": None, "etcdbackupdata": None}
+HOST_NAME = os.environ.get('RANCHER_HOST_NAME', "testcustom")
+
+rke_config["services"]["etcd"]["backupConfig"] = {
```

Comment:
Maybe we can put it in a common place and import than instead ? I see this used in other files like `test_bkp_restore_s3_with_creds.py`, etc

Reply:
`with_creds` and `with_iam` use different configs, I'm using the same as `with_creds` but this could allow us to be more flexible with buckets configurations if needed.